src/components/DefaultImage.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import React from 'react';
+
+type DefaultImageProps = {
+  /**
+   * The address of the image you want to display.
+   */
+  imageSrc: string;
+  /**
```

Comment:
In the comment for the defaultSrc property, I would mention that it's optional. The styleguidist stuff doesn't seem to do the best job with identification of default properties in the Rest components.